engine/core/test-src/org/pentaho/reporting/engine/classic/core/modules/gui/base/parameters/TextFieldParameterComponentTest.java

```diff
@@ -38,7 +38,6 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.pentaho.reporting.engine.classic.core.ResourceBundleFactory;
-import org.pentaho.reporting.engine.classic.core.modules.gui.base.ParameterReportControllerPane;
 import org.pentaho.reporting.engine.classic.core.parameters.ParameterAttributeNames;
```

Comment:
- import org.pentaho.reporting.engine.classic.core.modules.gui.base.ParameterReportControllerPane; is deleted while wingman still claim for

"/home/buildguy/project/engine/core/test-src/org/pentaho/reporting/engine/classic/core/modules/gui/base/parameters/TextFieldParameterComponentTest.javaorg.pentaho.reporting.engine.classic.core.modules.gui.base.ParameterReportControllerPane. at Line: 41 Column: 8"

wingman shows no mercy as usually.